frontend/components/app.js

```diff
@@ -46,48 +27,75 @@ class App extends React.Component {
   }
 
   render() {
-    const { children } = this.props;
-    const storeState = this.state;
+    const { user, children, notifications } = this.props;
 
-    if (storeState.user.isLoading) {
+    if (!user || user.isLoading) {
```

Comment:
The [user reducer sets the state to `false` if the user fetch fails](https://github.com/18F/federalist/blob/staging/frontend/reducers/user.js#L17-L19). This means that for an unauthenticated user, `this.props.user` will be false.

When there is an authed user, the `App` component renders the `Home` component at its index. The `Home` component [redirects to the site list when it finds a logged in user](https://github.com/18F/federalist/blob/staging/frontend/components/home.js#L18). 

But, without an authed user, the `App` component expects to render the `Home` component, and for the `Home` component to display our fancy marketing content. It looks like this change breaks that by having the App component perpetually render the loading indicator if the user is not logged in.